Python/Tests/ReplWindowUITests/ReplWindowPythonTests.cs

```diff
@@ -78,6 +78,7 @@ public abstract class ReplWindowPythonTests : ReplWindowPythonSmokeTests {
         /// Type "raise Exception()", hit enter, raise Exception() should have
         /// appropriate syntax color highlighting.
         /// </summary>
+        [Ignore] // https://github.com/Microsoft/PTVS/issues/2762
```

Comment:
IIRC, there's a test category for tests we expect to fail right now (`ExpectedFail` or similar - will be skipped for CI runs but not locally)